python/nav/models/arnold.py

```diff
@@ -59,7 +65,11 @@ class Identity(models.Model):
     autoenablestep = models.IntegerField(null=True, default=2)
     mail = VarcharField(blank=True)
     organization = models.ForeignKey(
-        'Organization', on_delete=models.CASCADE, db_column='orgid', null=True
+        'Organization',
+        on_delete=models.CASCADE,
+        db_column='orgid',
+        null=True,
+        related_name="identities",
```

Comment:
My only nitpick here seems to be that the term "identities" is pretty vague outside the confines of the Arnold ecosystem (the default `identity_set` was no better).

I would argue that related names on models outside of the Arnold namespace should be be more qualified, something like `arnold_identities`.